.gitignore

```diff
@@ -51,6 +51,8 @@ coverage.xml
 
 # Sphinx documentation
 docs/_build/
+docs/doctrees/
+docs/html/
```

Comment:
I think this should be `docs/*/`. We don't want anything except the Makefile. In particular, I agree with Amy that we don't want `docs/source` in the repo -- it's a byproduct of the documentation build process.